web/src/beta/features/ProjectSettings/innerPages/PublicSettings/AliasSettings/EditPanel.tsx

```diff
@@ -80,9 +85,9 @@ const EditPanel: FC<AliasSettingProps> = ({
               <Typography size="body" color={theme.content.weak}>
                 {"https://"}
               </Typography>
-              <TextInput value={localAlias} onChange={handleChange} />
+              <TextInput value={localAlias} onBlur={handleChange} />
               <Typography size="body" color={theme.content.weak}>
-                {".visualizer.reearth.io"}
+                {`.${domain}`}
```

Comment:
Please refactor the solution for this input, reference `publicUrl` in `AliasSettings/index.tsx`